I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/custom/login.rpy", line 32, in script
    $say()
  File "game/custom/login.rpy", line 32, in <module>
    $say()
  File "game/script.rpy", line 246, in say
    term(desc + "\n" + append)
  File "renpy/common/00nvl_mode.rpy", line 314, in do_display
    **display_args)
  File "game/script.rpy", line 181, in update_input
    term("That's not very nice.")
  File "renpy/common/00nvl_mode.rpy", line 314, in do_display
    **display_args)
Exception: Cannot start an interaction in the middle of an interaction, without creating a new context.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/custom/login.rpy", line 32, in script
    $say()
  File "/Applications/renpy-6.99.12.4-sdk/renpy/ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/custom/login.rpy", line 32, in <module>
    $say()
  File "game/script.rpy", line 246, in say
    term(desc + "\n" + append)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/character.py", line 877, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "renpy/common/00nvl_mode.rpy", line 314, in do_display
    **display_args)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/character.py", line 508, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/ui.py", line 285, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/core.py", line 2526, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/core.py", line 3204, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/layout.py", line 960, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/layout.py", line 960, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/layout.py", line 960, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/screen.py", line 651, in event
    rv = self.child.event(ev, x, y, st)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/layout.py", line 960, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/layout.py", line 232, in event
    rv = d.event(ev, x - xo, y - yo, st)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/layout.py", line 960, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/behavior.py", line 1316, in event
    self.update_text(content, self.editable, check_size=True)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/behavior.py", line 1182, in update_text
    self.changed(new_content)
  File "game/script.rpy", line 181, in update_input
    term("That's not very nice.")
  File "/Applications/renpy-6.99.12.4-sdk/renpy/character.py", line 877, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "renpy/common/00nvl_mode.rpy", line 314, in do_display
    **display_args)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/character.py", line 508, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/ui.py", line 285, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/core.py", line 2504, in interact
    raise Exception("Cannot start an interaction in the middle of an interaction, without creating a new context.")
Exception: Cannot start an interaction in the middle of an interaction, without creating a new context.

Darwin-16.7.0-x86_64-i386-64bit
Ren'Py 6.99.12.4.2187
> turing 0.1
